### ISSUE TITLE
Pretty sure this is not serious

### ISSUE THREAD

**Issue by cowzy5** [Ha3MrX/DDos-Attack#27]
Sorry if my english fails, but im pretty sure this is a joke... And its not even a well built one, the code is filled with flaws, i had to fix everything... Besides... The loading bar is not actually loading anything... It just looks cool

**Comment by Lampese** [comment]
Yes,this code is hava many defect,but it may have stopped maintenance

**Comment by qequ** [comment]
that progress bar...LOL

**Comment by Lampese** [comment]
But it seems that every programmer can speak English? (in use translation exchange)
